cmd/deploy/deploy.go

```diff
@@ -328,19 +326,21 @@ func (d *deployCommand) processResourceDeploymentResponse(
 			}
 			return err
 		}
-		if resp.GetAck() {
-			if !resp.GetSuccess() {
-				d.logger.Error(resp.GetMessage())
+
+		if logStatus := resp.GetLogStatus(); logStatus != nil {
```

Comment:
we can move the log handling to a single place, this is duplicate code in all the commands

Reply:
let's doing it once it's merged on `simplify-observer-response` branch